coremltools/converters/mil/mil/ops/defs/control_flow.py

```diff
@@ -97,6 +99,22 @@ def value_inference(self):
 
 @register_op(doc_str="")
 class const(Operation):
+    """
+    Return constant values represented as ``ImmediateValue`` or ``FileValue``
```

Comment:
Please change to : "Return constant values , represented as .....in the MIL format"